updateclang/clang.symbols

```diff
@@ -13,8 +13,11 @@
 
 ###
 
+--include-typedef CXCursor
 --include-typedef CXCursorVisitor
 --include-typedef CXIndex
+--include-typedef CXSourceRange
```

Comment:
Interesting side-effect of the new translation strategy: since struct A depends on struct B in its layout, if you need to generate A now you also need to generate B. Otherwise the generated code will have an unresolved reference to B from A's layout.